Gemfile

```diff
@@ -2,7 +2,8 @@
 source "https://rubygems.org"
 
 gem "concurrent-ruby", "~> 1.0.0"
-gem "flay"
+gem "flay", "~> 2.10"
+gem "sexp_processor", "~> 4.10" # TODO: Remove when flay >= 2.10
```

Comment:
Yeah it can come out... But... Won't hurt to leave it in since it will trigger a full rebuild of node5.